.github/workflows/tests.yml

```diff
@@ -120,11 +129,13 @@ jobs:
         run: bash test/test_workflow_local_with_singularity.sh
 
       - name: run local test with Conda
-        run: bash test/test_workflow_local_with_conda.sh
+        run: |
+          conda config --set channel_priority strict
```

Comment:
What's the reason behind changing the channel priority here? If I see this right, one set of tests runs with `true` and the other with `strict`, right? Why?